internal/indexer/balances.go

```diff
@@ -0,0 +1,72 @@
+package indexer
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/Taraxa-project/taraxa-indexer/internal/chain"
+	"github.com/Taraxa-project/taraxa-indexer/internal/utils"
+	"github.com/Taraxa-project/taraxa-indexer/models"
+	log "github.com/sirupsen/logrus"
+)
+
+func UpdateBalancesInternal(ptr *[]models.Account, trx models.Transaction) (err error) {
```

Comment:
no need to pass array as pointer here

Reply:
After the change of Account.Balance to *big.Int I think methods from this file could be replaced with something like 
```
ModifyBalance(address string, amount *big.Int) {...}

ModifyBalance(from, big.NewInt(0).Neg(amount))
ModifyBalance(to, amount)
```

Reply:
Hmm, I think if we don't pass as pointer here doesn't that mean that we copy the array in the function scope for every call? I'd like to avoid that

Reply:
https://go.dev/tour/moretypes/8